x-pack/plugins/ml/public/application/services/ml_api_service/jobs.ts

```diff
@@ -48,8 +48,8 @@ export const jobsApiProvider = (httpService: HttpService) => ({
     });
   },
 
-  jobs(jobIds: string[]) {
-    const body = JSON.stringify({ jobIds });
+  jobs(jobIds: string[], excludeGenerated?: boolean) {
+    const body = JSON.stringify({ jobIds, excludeGenerated });
```

Comment:
as `excludeGenerated ` could be undefined, it should be optionally added to the body

Reply:
Removed here as it's no longer needed 
 https://github.com/elastic/kibana/pull/88898/commits/e9ad8f16c6e349a3d7bbc5199ad6d12de0f1bb72